ts/src/kucoin.ts

```diff
@@ -1111,7 +1160,13 @@ export default class kucoin extends Exchange {
                 'settleId': undefined,
                 'type': 'spot',
                 'spot': true,
-                'margin': this.safeBool (market, 'isMarginEnabled'),
+                'crossMargin': hasCrossMargin,
```

Comment:
can e remove these fields crossMargin/marginMode/etc? we should add them in a different PR where we update the types as well

Reply:
uh oh, they were leftovers, we are not going to implement those fields at all

Reply:
pushed an update, check it now

Reply:
ah wait, you mean to remove marginMode too? it would require changing implementation of fetchMarkets back to original stage, and then there wouldnt be much left in this PR at all.
maybe I can open a separate PR to add types at first?